src/components/reviewMatches/SearchModal.tsx

```diff
@@ -22,17 +22,28 @@ const options: ToggleButtonOptions[] = [
   { value: 1, label: 'Simple search' }
 ]
 
-const SearchModal: FC<{ isOpen: boolean; onClose: () => void }> = ({
-  isOpen,
-  onClose
-}) => {
+const SearchModal: FC<{
+  isOpen: boolean
+  onClose: () => void
+  onChange: (query: SearchQuery | CustomSearchQuery | undefined) => void
+}> = ({ isOpen, onClose, onChange }) => {
   const [tabValue, setTabValue] = useState('Custom search')
   const [refineSearchQuery, setRefineSearchQuery] = useState<
     SearchQuery | CustomSearchQuery | undefined
   >(undefined)
 
+  const handleClose = () => {
+    onClose()
+    setTabValue('Custom search')
```

Comment:
For "Custom search" and "Simple search", I think we can declare a enum type if it's not already there.